wgsl/index.bs

```diff
@@ -553,7 +553,7 @@ An attribute must not be specified more than once per object or type.
 <div class='syntax' noexport='true'>
   <dfn for=syntax>attribute</dfn> :
 
-    | [=syntax/attr=] [=syntax/ident=] [=syntax/paren_left=] ( [=syntax/literal_or_ident=] [=syntax/comma=] ) * [=syntax/literal_or_ident=] [=syntax/paren_right=]
+    | [=syntax/attr=] [=syntax/ident=] [=syntax/paren_left=] ( [=syntax/literal_or_ident=] [=syntax/comma=] ) * [=syntax/comma=] ? [=syntax/literal_or_ident=] [=syntax/paren_right=]
```

Comment:
I think if the intent is to allow `@x(x, y, z,)` (notice the last comma), this should instead be:

```
[=syntax/attr=] [=syntax/ident=] [=syntax/paren_left=] ( [=syntax/literal_or_ident=] [=syntax/comma=] ) * [=syntax/literal_or_ident=] [=syntax/comma=] ? [=syntax/paren_right=]
```

Thanks!

Reply:
Thanks for the careful check! Done.

Reply:
Thanks!